### PR TITLE
Fix error messages in parsing of bit commands

### PR DESCRIPTION
**Pull request by gogim1**
This PR makes the error message consistent with the error message in Redis when calling the bit commands.

### CODE REVIEW

File: src/commands/cmd_bit.cc
Comment (mapleFU) [inline src/commands/cmd_bit.cc:160]: Seems that Redis would first check if it's integer ( via `getLongFromObjectOrReply`) and then check it's `1` or `0`

```c
    /* Parse the bit argument to understand what we are looking for, set
     * or clear bits. */
    if (getLongFromObjectOrReply(c,c->argv[2],&bit,NULL) != C_OK)
        return;
    if (bit != 0 && bit != 1) {
        addReplyError(c, "The bit argument must be 1 or 0.");
        return;
    }
```

Can we keep the same?

**Review by mapleFU — COMMENTED** [review]
Nice catch!

File: src/commands/cmd_bit.cc
Comment (gogim1) [reply src/commands/cmd_bit.cc:160]: done

**Review by mapleFU — DISMISSED** [review]

**Review by mapleFU — DISMISSED** [review]

File: src/commands/cmd_bit.cc
Comment (PragmaTwice) [inline src/commands/cmd_bit.cc]:
```suggestion
    if (*parse_arg == 0) {
```

File: src/commands/cmd_bit.cc
Comment (PragmaTwice) [inline src/commands/cmd_bit.cc]: ditto

Comment (gogim1) [reply src/commands/cmd_bit.cc]: done

**Comment by mapleFU** [comment]
Will merge if all CI passed

**Review by git-hulk — APPROVED** [review]

**Review by mapleFU — APPROVED** [review]

**Comment by mapleFU** [comment]
Thank you for catching this @GoGim1

**Comment by PragmaTwice** [comment]
I want to complain about the PR title, which is not formal and clear enough to me. A better message may be **"Fix error messages in parsing of bit commands"**.

@mapleFU You can adjust the commit message before you merge it, please do not miss the chance.

And please do not use spoken English and abbreviation, and avoid grammatical errors in command messages.

**Comment by mapleFU** [comment]
Sorry for that, I'll wait for some time before merging. You can force push to overwrite the title @PragmaTwice

**Comment by PragmaTwice** [comment]
It is forbidden by project settings to force push to the unstable branch.

Just be careful next time : )

**Pull request merged**
